test/system/test_tpm2_createprimary_all.sh

```diff
@@ -52,3 +52,15 @@ for gAlg in 0x04 0x0B 0x0C 0x0D 0x12
         done
 done
 
+#test for createprimary objects with policy authorization structures
+echo "f28230c080bbe417141199e36d18978228d8948fc10a6a24921b9eba6bb1d988" \
+| xxd -r -p > policy.bin
+tpm2_createprimary  -A o -G 0x1 -g 0xb -E -C prim.ctx -L policy.bin
+tpm2_readpublic -c prim.ctx -o testprim.pub
+cmp -i 14:0 -n 32 testprim.pub policy.bin -s
```

Comment:
Do we want this to fail this far, or should we check rc codes for each command? I would imagine each command, so we can have better output messages on error.